### ISSUE TITLE
Having trouble running pacoloco via Docker

### ISSUE THREAD

**Issue by 987computer** [anatol/pacoloco#50]
When I try to start the Docker container after building it with the provided instructions (as well as putting the config file listed below), this is what it returns:

**pacoloco.go:102: Reading config file from /etc/pacoloco.yaml
pacoloco.go:105: read /etc/pacoloco.yaml: is a directory**

**Comment by anatol** [comment]
cc @ilya-zlobintsev

**Comment by anatol** [comment]
/etc/pacoloco.yams is a config file. It must not be a directory.

**Comment by ilya-zlobintsev** [comment]
Are you sure that you've created the config file? By default if the host path specified in a volume doesn't exist, docker will create it as a directory.

**Comment by 987computer** [comment]
Yeah. In fact, I already created it beforehand getting that error. Trying the same steps with that said config file in the "etc" directory returns the same result.

**Comment by cocoonkid** [comment]
@987computer  Did you ever solve this?   Docker works perfectly for me here.

**Comment by Focshole** [comment]
> When I try to start the Docker container after building it with the provided instructions (as well as putting the config file listed below), this is what it returns:
> 
> **pacoloco.go:102: Reading config file from /etc/pacoloco.yaml pacoloco.go:105: read /etc/pacoloco.yaml: is a directory**
Try [logging into the docker container](https://stackoverflow.com/a/73944958/4746441) , then check what is going on in `/etc/pacoloco.yaml`

**Comment by HarHarLinks** [comment]
docker works perfectly here.

@987computer: you appear to not have created the yaml file before starting the container and/or in the wrong place. Docker will create an empty directory of that name in those cases, which leads to this error.

1. You need to stop and any existing container.
2. Create the `pacoloco.yaml` file at a reasonable place on your host system. I use `/opt/pacoloco/pacoloco.yaml` (next to /opt/pacoloco/cache`).
3.  Then look at where the example command or compose file says `/path/to/config/pacoloco.yaml:/etc/pacoloco.yaml`.
4. Set the left part of the argument to the path on your host. In my example that means you get `/opt/pacoloco/pacoloco.yaml:/etc/pacoloco.yaml`.
5. Run.

**Comment by Focshole** [comment]
Shall we close this issue?

**Comment by fhteagle** [comment]
Same issue using docker-compose. In docker-compose.yaml, 

``` 
...
volumes:
- ./pacoloco/pacoloco.yaml:/etc/pacoloco.yaml
...
```

the pacoloco.yaml is not interpreted as a single file, but as a directory. This prevents pacoloco from starting, and sends it into a start loop.

Instead of putting the config file in /etc/ itself, it would be better if it was in /etc/pacoloco/pacoloco.yaml . It would be a breaking change if it was only there, so pacoloco could look at both locations to find the config file.

**Comment by adrfantini** [comment]
I also had this issue when compising using `dockge`, but recreating the stack from scratch worked.

Surely this can be closed!
